s4e-backend/src/main/java/pl/cyfronet/s4e/controller/AppUserController.java

```diff
@@ -143,8 +151,23 @@
             @ApiResponse(code = 200, message = "User was added")
     })
     @PostMapping("/institutions/{institution}/users")
-    public ResponseEntity<?> addUserToInstitution(@RequestBody @Valid CreateUserWithGroupsRequest request) {
-
+    public ResponseEntity<?> addUserToInstitution(@RequestBody @Valid CreateUserWithGroupsRequest request,
+                                                  @PathVariable("institution") String institutionSlug) {
+        AppUser appUser = AppUser.builder()
+                .name(request.getName())
+                .surname(request.getSurname())
+                .email(request.getEmail())
+                .password(passwordEncoder.encode("passTemporary"))
```

Comment:
Maybe better, if it was random. Unless it is only for the demo?

Reply:
only for demo

Reply:
Add a FIXME there saying to fix it later